### ISSUE TITLE
PAPI error

### ISSUE THREAD

**Issue by patrikkoudelka** [IntellectualSites/PlotSquared#4501]
### Server Implementation

Paper

### Server Version

1.21.1

### Describe the bug

Error printed in console with PAPI

### To Reproduce

The error apears after join

### Expected behaviour

Without errors

### Error log (if applicable)

https://paste.gg/p/anonymous/f4666b5a8a384f2ab2e08cc788270c0c

### Plot Debugpaste

https://athion.net/ISPaster/paste/view/e1f09bcf2a3f4fff9de9afc45234d11b

### PlotSquared Version

 7.3.11

### Checklist

- [X] I have included a Plot debugpaste.
- [X] I am using the newest build from https://www.spigotmc.org/resources/77506/ and the issue still persists.

### Anything else?

---

**Comment by RedstoneFuture** [comment]
I can't replicate it right now with `PlaceholderAPI-2.11.6` and `LuckPerms-Bukkit-5.4.141` in a minimalistic setup. Maybe a behavior from the PAPI DEV-version or other plugins?

**Comment by patrikkoudelka** [comment]
> I can't replicate it right now with `PlaceholderAPI-2.11.6` and `LuckPerms-Bukkit-5.4.141` in a minimalistic setup. Maybe a behavior from the PAPI DEV-version or other plugins?

Hello, and have you had also extracontexts? Which is addon of the luckperms. And use hasplot placeholder here.

**Comment by RedstoneFuture** [comment]
> > I can't replicate it right now with `PlaceholderAPI-2.11.6` and `LuckPerms-Bukkit-5.4.141` in a minimalistic setup. Maybe a behavior from the PAPI DEV-version or other plugins?
> 
> Hello, and have you had also extracontexts? Which is addon of the luckperms. And use hasplot placeholder here.

I don't use [ExtraContexts](https://github.com/LuckPerms/ExtraContexts) on my servers, but I know it. Do you get valid results with the `/papi parse <player> <placeholder>` command?

**Comment by patrikkoudelka** [comment]
Yes, I do. And the error is mainly from extracontexts and the placeholder itself. That's why I am asking.
